### PR TITLE
feat(replays): Add additional filterable fields

### PR DESCRIPTION
**Pull request by cmanallen**
closes: https://github.com/getsentry/replay-backend/issues/242

### CODE REVIEW

File: src/sentry/replays/lib/query.py [hunk 1]
```diff
@@ -152,8 +155,20 @@ def _has_condition(
         if isinstance(value, list):
             return self._has_any_condition(Op.IN if operator == Op.EQ else Op.NOT_IN, value)
 
+        if self.is_uuid:
+            # Client side UUID validation.  If this fails we use a condition which is always
+            # false.  E.g. 1 == 2.  We don't use toUUIDOrZero because our Clickhouse version does
+            # not support it.
```
Comment (JoshFerge) [inline src/sentry/replays/lib/query.py:161]: why not just raise an error here?

**Review by JoshFerge — COMMENTED** [review]
looks like a test is failing other than that just Q about uuid condition

File: src/sentry/replays/lib/query.py (hunk 1 above)
Comment (cmanallen) [reply src/sentry/replays/lib/query.py:161]: @JoshFerge UX.  Do you have a preference between a 400 error and a falsey query condition?

Comment (JoshFerge) [reply src/sentry/replays/lib/query.py:161]: hrmmm, good point. with the search bar 400s aren't a great experience. yeah this is good!

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**